[Wishlist] facilite inheritance to create picking from pos

celm1990 opened the pull request:
**Description of the issue/feature this PR addresses:**
On some country or custom modules is need pass values before create picking, this PR has as objective facilite inheritance
**Current behavior before PR:**

**Desired behavior after PR is merged:**




--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

celm1990 commented:
@mart-e Sorry for ping, can you helpme with this PR please

mart-e commented:
Hi,

Thanks for the PR, I have changed the commit to follow the [commit message guidelines](https://www.odoo.com/documentation/12.0/reference/guidelines.html#commit-message-structure) and changed a bit the indentation but it looks good.

@robodoo r+

celm1990 commented:
> Hi,
> 
> Thanks for the PR, I have changed the commit to follow the [commit message guidelines](https://www.odoo.com/documentation/12.0/reference/guidelines.html#commit-message-structure) and changed a bit the indentation but it looks good.
> 
> @robodoo r+

thanks so much @mart-e
